docs/source/conventions.md

```diff
@@ -73,4 +73,4 @@ Rover commands that take a file path as an option can instead accept input from
 rover graph introspect http://localhost:4000 | rover graph check my-graph --schema -
```

Comment:
Thoughts on making this a multi-line command so that the `-` is more visible?

```suggestion
rover graph introspect http://localhost:4000 \
  | rover graph check my-graph --schema -
```